src/views/2_SessionComponent.tsx

```diff
@@ -25,6 +25,10 @@ export class SessionComponent extends React.Component<Props, {}> {
     }
 
     componentDidMount() {
+        this.session.onscroll = () => {
+            const shutterOffset = this.session.scrollTop;
+            this.shutter.style.transform = `translateY(${shutterOffset}px)`;
+        };
```

Comment:
I tried implementing a few things, but none of them worked. This is how that old PR does it:

https://jsfiddle.net/7rvxpoaj/1/

But it doesn't work. I also tried some other stuff and couldn't make it work.